great_expectations/expectations/expectation.py

```diff
@@ -1811,11 +1811,6 @@ def validate_metric_value_between_configuration(
         except AssertionError as e:
             raise InvalidExpectationConfigurationError(str(e))
 
-        if min_val is not None and max_val is not None and min_val > max_val:
-            raise InvalidExpectationConfigurationError(
-                "Minimum Threshold cannot be larger than Maximum Threshold"
-            )
```

Comment:
Why remove this?

Reply:
@NathanFarmer Great question -- this check is actually a bug.  It compares string-valued datetimes lexicographically, which, of course, fails when TimeZone is part of the datetime/timestamp string.  The relevant checks are performed sufficiently adequately downstream.  Thanks!